### PR TITLE
Rubocop -a on lib/jekyll

### LINKED ISSUE #5665: Jekyll has invalid formatting for deprecation message.
https://github.com/jekyll/jekyll/blame/master/lib/jekyll/collection.rb#L35
/cc @DirtyF

### PR DESCRIPTION
**Pull request by DirtyF**
Fix #5665

### CODE REVIEW

File: lib/jekyll/collection.rb [hunk 1]
```diff
@@ -208,14 +209,15 @@ def read_document(full_path)
...
+
     def read_static_file(file_path, full_path)
       relative_dir = Jekyll.sanitized_path(
         relative_directory,
         File.dirname(file_path)
       ).chomp("/.")
 
       files << StaticFile.new(site, site.source, relative_dir,
-        File.basename(full_path), self)
+                              File.basename(full_path), self)
```
Comment (ashmaroli) [inline lib/jekyll/collection.rb]: whoa! rubocop did this?
may I suggest: ?
```ruby
files << StaticFile.new(
  site,
  site.source,
  relative_dir, 
  File.basename(full_path),
  self
)
```

File: lib/jekyll/excerpt.rb [hunk 2]
```diff
@@ -30,8 +30,7 @@ def data
       @data
     end
 
-    def trigger_hooks(*)
-    end
+    def trigger_hooks(*); end
```
Comment (ashmaroli) [inline lib/jekyll/excerpt.rb]: jus curious. why do we have an empty method?

Comment (pathawks) [inline lib/jekyll/excerpt.rb]: So it can be overridden :+1:

**Comment by ashmaroli** [comment]
noticed this just now.. PRs shouldn't have a `bug` label.. that's better for issue tickets. PRs should be sporting a `bug-fix` label instead.. :smiley:

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Pull request merged**